proto/spacemesh/v1/smesher_types.proto

```diff
@@ -77,49 +69,42 @@ enum ComputeApiClass {
     COMPUTE_API_CLASS_VULKAN = 3;
 }
 
-// Basic post data. Used for users to provide requested post data and by
-// other messages which contain post data.
-message PostData {
-    string path = 1; // User provided path to create the post data files at
-    uint64 data_size = 2; // Requested post data size
-    bool   append = 3; // Append to existing files if they exist. Otherwise overwrite.
-    bool   throttle = 4; // Throttle down setup phase computations while user is interactive on system
-    uint32 provider_id = 5; // A PostProvider id
+// Post init options. Used to define the requested options and by
+// other messages which contain the options.
+message PostInitOpts {
+    string data_dir = 1; // User provided path to create the post data files at
+    uint32 num_units = 2; // Number of PoST data commitment units to generate
+    uint32 num_files = 3; // Number of files to equally distribute the labels among
+    uint32 compute_provider_id = 4; // A PostProvider id
+    bool throttle = 5; // Throttle down setup phase computations while user is interactive on system
```

Comment:
Couldn't this be an int, to allow more fine-tuned throttling? E.g., max CPU percentage?

Reply:
It's referring to the utilization of GPU cores, not CPU time. The API complies with the internal API of `gpu-post` lib, so we should consider changing it there first.

Reply:
It's referring to the utilization of GPU cores, not CPU time. The API complies with the internal API of [`gpu-post`](https://github.com/spacemeshos/gpu-post) lib, so we should consider changing it there first.

Reply:
For the release of gpu-post lib for post it is indeed a boolean and it is not going to change to an int. Let's consider this feature for next release. We need to finalize and ship and we shouldn't change features so late in the cycle.